libraries/AP_HAL_ChibiOS/hwdef/FlywooF745/README.md

```diff
@@ -104,7 +126,7 @@ The GOKU GN 745 AIO does not have a builtin compass but it does have an external
 
 The board includes a NeoPixel LED pad.
 
-## Loading Firmware (you will need to compile your own firmware)
+## Loading Firmware
```

Comment:
a line pointing to fimrware.ardupilot.org including what folder name the firmware is located in would be helpful here